scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -377,15 +350,21 @@ source "${USHgfs}/preamble.sh"
   rm -f out_grd.* # Remove large binary grid output files
 
   cd $DATA
-
-
-  if [ "$fhr" = "$fhrg" ]
-  then
+ 
+  if [ "$fhr" = "$fhrg" ]; then
```

Comment:
Also - since this is now per forecast hour, I'm not sure what the point of this if statement is because on line 220: 

```
fhr=$(( 10#${FHR3} ))
  fhrg=$fhr 
```